web/app/view/UserDialogController.js

```diff
@@ -15,7 +15,7 @@
  */
 
 Ext.define('Traccar.view.UserDialogController', {
-    extend: 'Traccar.view.BaseEditDialogController',
+    extend: 'Traccar.view.ServerDialogController',
```

Comment:
This is a bit confusing. Maybe it should be renamed from `ServerDialogController` to something like `MapPickerDialogController`.